waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,39 @@
+{:kind "ReplicaSet"
+ :apiVersion #waiter/param :replicaset-api-version
+ :metadata {:name #waiter/param :k8s-name
+            :labels {:app #waiter/param :k8s-name
+                     :managed-by "waiter"}
```

Comment:
This value should come from the Clojure code as it is used in the queries too. Or we should have a comment here that specifies that this value should not be changed.

Reply:
Done.